modules/phb-2024/module.json

```diff
@@ -0,0 +1,68 @@
+{
+	"id": "phb-2024",
+	"name": "phb-2024",
+	"title": "Player’s Handbook (2024)",
+	"description": "Free product that provides listing content only.",
+	"authors": [
+		{
+			"name": "MrPrimate",
+			"email": "jack@mrprimate.co.uk",
+			"url": "https://mrprimate.co.uk",
+			"patreon": "https://www.patreon.com/MrPrimate",
+			"discord": "https://discord.gg/WzPuRuDJVP"
+		},
+		{
+			"name": "The Forge",
+			"url": "https://forge-vtt.com",
+			"email": "contact@forge-vtt.com",
+			"discord": "https://forge-vtt.com/discord",
+			"reddit": "https://www.reddit.com/r/ForgeVTT",
+			"twitter": "@ForgeVTT"
+		}
+	],
+	"url": "https://www.dndbeyond.com/sources/phb-2024",
+	"license": "",
+	"readme": "",
+	"bugs": "",
+	"changelog": "",
+	"flags": {},
+	"version": "0.0.0",
+	"compatibility": {
+		"minimum": "0.7.9",
+		"verified": "10"
+	},
+	"minimumCoreVersion": "0.7.9",
+	"compatibleCoreVersion": "10",
```

Comment:
Just noticing this; I assume these compatibility values are automatically generated by the scripts?

Have we actually checked compatibility with 0.7 (or 0.8, or v9) recently? I also wonder if we should increase the compatible number to 11 or even 12? We do have the system compatibility at 2.0.3+, which is for v10, so maybe that means we should leave compatible at 10.